protoc-gen-swagger/genswagger/template.go

```diff
@@ -518,14 +518,24 @@ func renderEnumerationsAsDefinition(enums enumMap, d swaggerDefinitionsObject, r
 
 // Take in a FQMN or FQEN and return a swagger safe version of the FQMN
 func fullyQualifiedNameToSwaggerName(fqn string, reg *descriptor.Registry) string {
+	ret, _ := fullyQualifiedNameToSwaggerNameOk(fqn, reg)
+	return ret
+}
+
+// Take in a FQMN or FQEN and return a swagger safe version of the FQMN and
+// a boolean indicating if FQMN was properly resolved.
+// TODO utrack: bad name?
```

Comment:
Just remove the other one and use the one that reports if it was found everywhere. It's a good change :+1:.